client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -21,6 +15,9 @@ import isRegeneratingJetpackPostByEmail from 'calypso/state/selectors/is-regener
 import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PressThis from '../press-this';
+import { PostByEmailSetting } from './post-by-email';
+import { PostByVoiceSetting } from './post-by-voice';
```

Comment:
```suggestion
import PressThis from 'calypso/my-sites/site-settings/press-this';
import { PostByEmailSetting } from 'calypso/my-sites/site-settings/publishing-tools/post-by-email';
import { PostByVoiceSetting } from 'calypso/my-sites/site-settings/publishing-tools/post-by-voice';
```
Nit, but let's be consistent with absolute import paths.

Reply:
Shoot! I made a point of fixing that, because it was mentioned on a previous PR. I must have lost it during a rebase 🤦